### ISSUE TITLE
:download: issue

### ISSUE THREAD

**Issue by adreyfus** [sphinx-contrib/confluencebuilder#393]
Hi,

I encounter a bug with the instruction :download:

The following code is valid for Sphinx, but not for confluencebuilder :

```
Télécharger :download:`à cette adresse <https://google.fr>`
```

**Comment by adreyfus** [comment]
Then I have the error : 

`"Error parsing xhtml: Unexpected close tag </xml>; expected </p>.\n `

**Comment by jdknight** [comment]
@adreyfus, thanks for reporting this issue. I was able to make a quick fix in #396 and have merged the change into the main branch. There should be a release soon (good timing; see #362) which should include the fix.

**Comment by adreyfus** [comment]
Wonderful ! Thank you so much !
